src/engraving/style/styledef.h

```diff
@@ -1,1500 +1,1521 @@
+    user12FramePadding,
+    user12FrameWidth,
+    user12FrameRound,
+    user12FrameFgColor,
+    user12FrameBgColor,
+
+    letRingFontFace,
+    letRingFontSize,
+    letRingLineSpacing,
+    letRingFontSpatiumDependent,
+    letRingFontStyle,
+    letRingColor,
+    letRingTextAlign,
+    letRingHookHeight,
+    letRingPlacement,
+    letRingPosAbove,
+    letRingPosBelow,
+    letRingLineWidth,
+    letRingLineStyle,
+    letRingBeginTextOffset,
+    letRingText,
+    letRingFrameType,
+    letRingFramePadding,
+    letRingFrameWidth,
+    letRingFrameRound,
+    letRingFrameFgColor,
+    letRingFrameBgColor,
+    letRingEndHookType,
+
+    palmMuteFontFace,
+    palmMuteFontSize,
+    palmMuteLineSpacing,
+    palmMuteFontSpatiumDependent,
+    palmMuteFontStyle,
+    palmMuteColor,
+    palmMuteTextAlign,
+    palmMuteHookHeight,
+    palmMutePlacement,
+    palmMutePosAbove,
+    palmMutePosBelow,
+    palmMuteLineWidth,
+    palmMuteLineStyle,
+    palmMuteBeginTextOffset,
+    palmMuteText,
+    palmMuteFrameType,
+    palmMuteFramePadding,
+    palmMuteFrameWidth,
+    palmMuteFrameRound,
+    palmMuteFrameFgColor,
+    palmMuteFrameBgColor,
+    palmMuteEndHookType,
+
+    fermataPosAbove,
+    fermataPosBelow,
+    fermataMinDistance,
+
+    fingeringPlacement,
+
+    articulationMinDistance,
+    fingeringMinDistance,
+    hairpinMinDistance,
+    letRingMinDistance,
+    ottavaMinDistance,
+    palmMuteMinDistance,
+    pedalMinDistance,
+    repeatMinDistance,
+    textLineMinDistance,
+    systemTextLineMinDistance,
+    trillMinDistance,
+    vibratoMinDistance,
+    voltaMinDistance,
+    figuredBassMinDistance,
+    tupletMinDistance,
+
+    autoplaceEnabled,
+    usePre_3_6_defaults,
+    defaultsVersion,
+
+    STYLES
+    ///\}
+};
+END_QT_REGISTERED_ENUM(Sid)
+
+inline uint qHash(Sid id)
+{
+    return static_cast<uint>(id);
+}
+
+//---------------------------------------------------------
+//   StyledProperty
+///   \cond PLUGIN_API \private \endcond
+//---------------------------------------------------------
+struct StyledProperty {
+    Sid sid;
+    Pid pid;
+};
+
+typedef std::vector<StyledProperty> ElementStyle;
+
+//---------------------------------------------------------
+//   StyleDef
+///   \cond PLUGIN_API \private \endcond
+//---------------------------------------------------------
+struct StyleDef
+{
+private:
+
+    friend class MStyle;
+
+    struct StyleValue {
+        Sid _idx;
+        const char* _name;         // xml name for read()/write()
+        QVariant _defaultValue;
+
+    public:
+        Sid  styleIdx() const { return _idx; }
+        int idx() const { return int(_idx); }
+        const char* valueType() const { return _defaultValue.typeName(); }
+        const char* name() const { return _name; }
+        const QVariant& defaultValue() const { return _defaultValue; }
+    };
+
+    static const StyleValue styleValues[int(Sid::STYLES)];
+};
+}
+
+#endif // MU_ENGRAVING_STYLEDEF_H
```

Comment:
What happened here, maybe just a carriage return / linefeed change? If so: please revert.

Reply:
I am sorry I do not understand. If you are asking if I have made any change to this file that pertains to the project, then yes. But the difference should be few lines only. I do not know what is happening here.

Reply:
Then please double check in those changes. I don't see a need to change the entire file, this will cause merge conflicts later

Reply:
Done